apps/planka/docker-compose.yml

```
@@ -0,0 +1,70 @@
+version: '3'
+
+services:
+  planka:
+    image: ghcr.io/plankanban/planka:1.12.0
+    container_name: planka
+    command: >
+      bash -c
+        "for i in `seq 1 30`; do
+          ./start.sh &&
+          s=$$? && break || s=$$?;
+          echo \"Tried $$i times. Waiting 5 seconds...\";
+          sleep 5;
+        done; (exit $$s)"
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data/user-avatars:/app/public/user-avatars
+      - ${APP_DATA_DIR}/data/project-background-images:/app/public/project-background-images
+      - ${APP_DATA_DIR}/data/attachments:/app/private/attachments
+    ports:
+      - ${APP_PORT}:1337
+    environment:
+      BASE_URL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}/
```

By removing the trailing slash, the app can be launched correctly
```suggestion
      BASE_URL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}
```

---

It does not exactly work... I have set my listen ip to 0.0.0.0 and it will only actually load from http://0.0.0.0:8016 rather than for example http://localhost:8016 or http://ip:8016